next-frontend/src/globals/About.ts

```diff
@@ -0,0 +1,98 @@
+import { GlobalConfig } from "payload";
+import { markdownConvertedField } from "@/collections/helpers";
+
+export const AboutUsPage: GlobalConfig = {
+  slug: "about-us-page",
+  label: "About Us Page",
+  access: {
+    read: () => true,
+  },
+  fields: [
+    {
+      name: "blocks",
+      type: "blocks",
+      required: true,
+      blocks: [
```

Comment:
You can pull the individual block choices out into their own `const` definitions using the `Block` type. Makes the nesting a little less deep and therefor easier to read

Reply:
Also, purely optional but more FYI: If you pass an `interfaceName` to the individual `Block`s, the codegen will expose that name when generating types, meaning you can reference the type of your blocks when typehinting.

This isn't immediately relevant for your current PR, because your blocks are relatively simple and you can pass around the individual "properties" of your blocks directly to the child components when rendering them.
But if you take a look at the `Home` page code, you will see that you can also pass down the block as a whole to child components using this `interfaceName` reference.